src/react/components/_dashboard/bConnected/NoBconnected.js

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { enableOAuth } from 'functions/user';
+
+import './NoBconnected.scss';
+
+const NoBconnected = ({ mode, actingAsUid, delegateActingAsUid, advisorActingAsUid, applicationLayer }) => {
+  const [showReminder, setShowReminder] = useState(false)
+
+  const toggleShowReminder = function() {
+    setShowReminder(!showReminder)
+  }
+
+  const className = 'NoBconnected';
+
+  return (
+    <div className={ className }>
+      {(mode === 'upnext' || mode === 'tasks') &&
+        <div>
+          {mode === 'tasks' &&
+            <span>Want to create your own personal bTasks?</span>
+          }
+          {mode === 'upnext' &&
+            <span>Want to see events from your bCal calendar?&nbsp;</span>
+          }
+          {!actingAsUid && !advisorActingAsUid && !delegateActingAsUid &&
```

Comment:
To read this line requires a lot of mental parsing. Could these conditional be extracted to a well-named constant?